services/pipeline/src/main/java/org/hyperledger/besu/services/pipeline/Pipeline.java

```diff
@@ -176,8 +176,7 @@ private Future<?> runWithErrorHandling(final ExecutorService executorService, fi
                 || t instanceof AsyncOperationException) {
               LOG.trace("Unhandled exception in pipeline. Aborting.", t);
             } else {
-              LOG.info("Unexpected exception in pipeline. Aborting.");
-              LOG.debug("Unexpected exception in pipeline. Aborting.", t);
+              LOG.info("Unexpected exception in pipeline. Aborting.", t);
```

Comment:
// log the stacktrace at info to help troubleshoot sync stuck issues

Reply:
maybe it should be at warn actually